apps/studio/src/components/CoreTabHeader.vue

```diff
@@ -138,7 +138,8 @@ import TabIcon from './tab/TabIcon.vue'
           { name: "Close Others", slug: 'close-others', handler: ({item}) => this.$emit('closeOther', item)},
           { name: 'Close All', slug: 'close-all', handler: ({item}) => this.$emit('closeAll', item)},
           { name: "Close Tabs to Right", slug: 'close-to-right', handler: ({item}) => this.$emit('closeToRight', item)},
-          { name: "Duplicate", slug: 'duplicate', handler: ({item}) => this.$emit('duplicate', item) }
+          { name: "Duplicate", slug: 'duplicate', handler: ({item}) => this.$emit('duplicate', item) },
+          { name: "Copy Name", slug: 'copy-name', handler: ({item}) => this.$emit('copyName', item) }
```

Comment:
Maybe rename to `Copy Entity Name`, and also grey this out if it's not a table or table property tab